src/resolver.ts

```diff
@@ -64,7 +64,7 @@ export const resolver =
         if (typeof schemaBase !== 'undefined') {
           const schema = rules.object().shape(schemaBase);
           // update args to the transformed ones by yup
-          args = await schema.validate(args);
+          args = await schema.validate(args, { context: ctx });
```

Comment:
Is `context` clear enough here since we're in yup land? Maybe rename this to something like `graphqlContext`?

Reply:
You can't change the name of the property. The second argument of `validate` has following type:

```ts
type Options = {
  strict: boolean;
  abortEarly: boolean;
  stripUnknown: boolean;
  recursive: boolean;
  context?: object;
}
```
([See yup docs](https://www.npmjs.com/package/yup#mixedvalidatevalue-any-options-object-promiseany-validationerror)).

The only thing that you could do is wrapping the `graphqlContext` in an object and the pass it to `validate`:

```ts
args = await schema.validate(args, { context: { graphlContext: ctx }});
```

Accessing would look like so:

```ts
yup.addMethod(yup.string, "currency", function () {
  return this.test(async (value, testContext) => {
    const { graphqlContext } = testContext.options.context
    // ...
  });
});
```

Not sure if that's improving things a lot, but I can live with both.

Let me know what you think 🙂